Dockerfile

```diff
@@ -1,9 +1,21 @@
 FROM ubuntu:21.04
 
-RUN apt-get update --yes && env DEBIAN_FRONTEND=noninteractive apt-get install openjdk-15-jdk maven --yes --no-install-recommends
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    openjdk-15-jdk \
+    maven && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
What does it do?

Reply:
Why remove it?

Reply:
> What does it do?   
@bajinsheng  sir command apt-get install -y --no-install-recommends openjdk-15-jdk maven installs the OpenJDK 15 and Maven packages on the Docker image. The -y flag automatically answers "yes" to any prompts during installation, and the --no-install-recommends flag tells apt-get not to install any recommended packages.  and Maven is a build automation tool used primarily for Java projects. This can help reduce the number of unnecessary packages and dependencies installed on the image, which can in turn help reduce the image size and decrease the risk of potential security vulnerabilities.

Reply:
> Why remove it?

And  "rm -rf /var/lib/apt/lists/*" removes the package lists downloaded by apt-get during the installation of the packages. These package lists are not needed after the packages have been installed, and removing them can significantly reduce the size of the Docker image .

Reply:
Thanks for your PR. However, I think the user permission and image size are not significant issues. These commands somehow increase the complexity.

Reply:
> Thanks for your PR. However, I think the user permission and image size are not significant issues. These commands somehow increase the complexity.

I understand your perspective and respect your decision. Thank you for considering my suggestions, and please let me know if there are any other changes or improvements you would like to see.